types/node/crypto.d.ts

```diff
@@ -128,19 +153,39 @@ declare module "crypto" {
         setAAD(buffer: Buffer | NodeJS.TypedArray | DataView, options?: { plaintextLength: number }): this;
     }
 
+    interface PrivateKeyInput {
+        key: string | Buffer;
+        format?: KeyFormat;
+        type?: 'pkcs1' | 'spki';
```

Comment:
```suggestion
        type?: 'pkcs1' | 'pkcs8' | 'sec1';
```
see https://nodejs.org/docs/latest-v11.x/api/crypto.html#crypto_crypto_createprivatekey_key